docs/content/latest/api/ysql/the-sql-language/statements/ddl_create_index.md

```diff
@@ -47,9 +47,57 @@ Use the `CREATE INDEX` statement to create an index on the specified columns of
 
 `CONCURRENTLY`, `USING method`, `COLLATE`, and `TABLESPACE` options are not yet supported.
 
+When an index is created on a populated table, YugabyteDB automatically
+backfills the existing data into the index.  In most cases, this uses an online
+schema migration.
+
+| | online | not online |
+| --- | --- | --- |
+| safe to do other DMLs during `CREATE INDEX`? | yes\* | no |
+| keeps other transactions alive during `CREATE INDEX`? | mostly | no |
+| parallelizes index loading? | yes | no |
+
+To disable online schema migration for YSQL `CREATE INDEX`, set the flag
+`ysql_disable_index_backfill=true` on **all** nodes and **both** master and
+tserver.  To disable online schema migration for one `CREATE INDEX`, use
+`CREATE INDEX NONCONCURRENTLY`.
+
+If online `CREATE INDEX` fails, it likely failed in the backfill step.  In that
+case, the index exists but is not usable.  Drop the index and try again.  If it
+still doesn't work,
+
+- did it time out?  Then, try increasing timeout flags:
+  - master `ysql_index_backfill_rpc_timeout_ms`
+  - tserver `ysql_wait_until_index_permissions_timeout_ms`
+- did you get "backfill failed to connect to DB"?  Then, you may be hitting an
+  issue with authentication.  If the version is <2.4 (stable) or <2.5.2
```

Comment:
Maybe what we want to say here is actually this: 
```suggestion
  issue with authentication. If you're on a stable version prior to v2.4, or a 2.5 version prior to 2.5.2,
```

Reply:
Cases like this really point out the trouble with out stable-vs-latest scheme of even/odd numbering :(